### DISCUSSION TITLE
"An unknown error occured" (or HTTP statuscode 413)

### DISCUSSION THREAD

**Discussion by waweic** [nextcloud/all-in-one#1015] [Questions] [answered]
In recently helped a friend installing Nextcloud AIO in a VM on Proxmox, with a reverse proxy (Nginx) in a different VM and the NEXTCLOUD_DATADIR env variable set to a mounted BTRFS subvolume.

When uploading any file larger than some MB, Nextcloud displays the error message "An unknown error occured". In the Firefox Network log, we can see a 413 response, from Apache, if I recall correctly.

Has anybody seen this before? Any ideas on what we could try? I figured that even tough it's probably a mistake in our configuration, it may not be a bad idea to have a solution to this in the documentation somewhere, if it's a common problem

**Comment by szaimen** [comment, marked as answer]
Hi, I suppose that you simply have not added the correct permissions. See https://github.com/nextcloud/all-in-one#how-to-change-the-default-location-of-nextclouds-datadir and 
![image](https://user-images.githubusercontent.com/42591237/184613798-0f011ccb-1d5f-4168-ad19-9e88e330326c.png)
